src/VisualStudio/Core/Def/ProjectSystem/RunningDocumentTableEventTracker.cs

```diff
@@ -183,23 +183,11 @@ public IEnumerable<(string moniker, ITextBuffer textBuffer, IVsHierarchy hierarc
 
         private IEnumerable<uint> GetInitializedRunningDocumentTableCookies()
         {
```

Comment:
Consider moving this too IVsRunningDocumentTableExtensions? You could then reuse it in the SolutionExplorerVerifierInProcess where you do the same checks again.

Reply:
Will leave like this for now, since I only really created the extension method as an `IEnumerable<uint>` adapter for a COM API.